libleak.c

```diff
@@ -164,8 +164,15 @@ static void lib_maps_build(void)
 			continue;
 		}
 
+		lib_maps = (struct lib_map *)realloc(lib_maps, (lib_map_num + 1) * sizeof(struct lib_map));
```

Comment:
The cast `(struct lib_map *)` is necessary for C++ but not for C. I think it's [bad](https://stackoverflow.com/a/605858/4794937) in fact. Please delete it.